src/renderer/api/endpoints/ingress.api.ts

```diff
@@ -107,6 +108,20 @@ export class Ingress extends KubeObject {
     }
     return ports.join(", ")
   }
+
+  getLoadBalancers() {
+    const ingressAddresses: string[] = [];
+    const { status: { loadBalancer } } = this;
+
+    if (!loadBalancer) return [];
+
+    loadBalancer.ingress.forEach(address => {
+      const entry = address.hostname ? address.hostname : address.ip;
+      ingressAddresses.push(entry); 
+    })
+
+    return ingressAddresses;
```

Comment:
I would suggest making this a map as it is much cleaner

```suggestion
    const { status: { loadBalancer = {} } } = this;

    return (loadBalancer.ingress ?? []).map(address => (
        address.hostname || address.ip
    ))
```

Reply:
That is much cleaner @Nokel81 - I'm still learning so having these examples is awesome to help me improve. One question, as `loadBalancer` could end up being an empty array are we good with the ingress and address property throwing up a warning in IDE validation? For example, in VS Code I get the following:

```
any
Property 'ingress' does not exist on type '{}'
```

Reply:
The above actually causes a CI/CD failure in integration tests so will see if I can resolve

Reply:
Amended as follows:

```
const { status: { loadBalancer = {ingress: []} } } = this;
```

This gives the loadBalancer an empty ingress object by default.

This ok?

Reply:
Yes that would be better